client/state/data-layer/wpcom/me/index.js

```diff
@@ -6,10 +6,12 @@ import devices from './devices';
 import notification from './notification';
 import settings from './settings';
 import sendVerificationEmail from './send-verification-email';
+import sites from './sites';
 
 export default mergeHandlers(
 	devices,
 	notification,
 	settings,
 	sendVerificationEmail,
+	sites
```

Comment:
I'd suggest to put a comma at the end of line here, so that whoever adding a new handler won't need to touch this line just for adding a comma. Just like this diff is benefited by `sendVerificationEmail,` here :) There are also plenty of other object expressions in this PR that we could consider to do the same.

However, this is more of a personal style preference, since it is not enforced in our style guide. Thus, please feel free to ignore it :)